Directory.Build.props

```diff
@@ -18,7 +18,8 @@
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('SOLARIS'))">Solaris</_hostOS>
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">windows</_hostOS>
     <_hostOS Condition="'$(TargetOS)' == 'Browser'">Browser</_hostOS>
-    <TargetOS Condition="'$(TargetOS)' == ''">$(_hostOS)</TargetOS>
+    <HostOS>$(_hostOS)</HostOS>
```

Comment:
a nit: we can find/replace _hostOS to HostOS in this file and delete this line. Line 13 sets the default.